recipes-ni/ni-sysd-workarounds/syswebserv-workaround.bb

```diff
@@ -0,0 +1,50 @@
+SUMMARY = "systemWebServer systemD workaround files"
+DESCRIPTION = "Workaround for systemWebServer systemD implementation"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SECTION = "base"
+
+SRC_URI = " \
+	file://syswebserv_workaround.service \
+	file://run_natinst.conf \
+"
+
+DEPENDS += " \
+	update-rc.d-native \
+	shadow-native \
+	pseudo-native \
+"
+
+inherit systemd
+SYSTEMD_PACKAGES = "${PN}"
+SYSTEMD_SERVICE:${PN} = "syswebserv_workaround.service"
+SYSTEMD_AUTO_ENABLE:${PN} = "enable"
+
+# Create syswebserv_workaround.service and install to /usr/lib/systemd/system
+FILES:${PN} += " \
+	${systemd_unitdir}system/syswebserv_workaround.service \
```

Comment:
Maybe just use `${systemd_system_unitdir}` instead of `${systemd_unitdir}system` everywhere in the file.